### PR TITLE
[zh] Tweak translation of CCM development page

### PR DESCRIPTION
**Pull request by tengqm**
The glossary shortcode is annoying. Sometimes it generates weird
sentence structures for Chinese.

### CODE REVIEW

File: content/zh/docs/tasks/administer-cluster/developing-cloud-controller-manager.md [hunk 1]
```diff
@@ -27,60 +27,59 @@ Since cloud providers develop and release at a different pace compared to the Ku
...
 
-### In Tree
+### 树内（In Tree）
 
 <!--
 For in-tree cloud providers, you can run the in-tree cloud controller manager as a {{< glossary_tooltip term_id="daemonset" >}} in your cluster. See [Cloud Controller Manager Administration](/docs/tasks/administer-cluster/running-cloud-controller/) for more details.
 -->
-对于 in-tree 驱动，你可以将 in-tree 云控制器管理器作为群集中的
-{{< glossary_tooltip term_id="daemonset" text="Daemonset" >}} 来运行。
+对于树内（In-Tree）驱动，你可以将树内云控制器管理器作为群集中的
```
Comment (howieyuen) [inline content/zh/docs/tasks/administer-cluster/developing-cloud-controller-manager.md:82]:
```suggestion
对于树内（In-Tree）驱动，你可以将树内云控制器管理器作为集群中的
```

**Review by howieyuen — CHANGES_REQUESTED** [review]

File: content/zh/docs/tasks/administer-cluster/developing-cloud-controller-manager.md (hunk 1 above)
Comment (tengqm) [reply content/zh/docs/tasks/administer-cluster/developing-cloud-controller-manager.md:82]: oops

File: content/zh/docs/tasks/administer-cluster/developing-cloud-controller-manager.md [hunk 2]
```diff
@@ -27,60 +27,59 @@ Since cloud providers develop and release at a different pace compared to the Ku
...
 
-### In Tree
+### 树内（In Tree）
 
 <!--
 For in-tree cloud providers, you can run the in-tree cloud controller manager as a {{< glossary_tooltip term_id="daemonset" >}} in your cluster. See [Cloud Controller Manager Administration](/docs/tasks/administer-cluster/running-cloud-controller/) for more details.
 -->
-对于 in-tree 驱动，你可以将 in-tree 云控制器管理器作为群集中的
-{{< glossary_tooltip term_id="daemonset" text="Daemonset" >}} 来运行。
+对于树内（In-Tree）驱动，你可以将树内云控制器管理器作为集群中的
```
Comment (ydFu) [inline content/zh/docs/tasks/administer-cluster/developing-cloud-controller-manager.md:82]:
```suggestion
对于树内（In-Tree）驱动，你可以将树内（In-Tree）云控制器管理器作为集群中的
```

**Review by ydFu — COMMENTED** [review]
Remarks for proper nouns should be the same remarks in an article.

File: content/zh/docs/tasks/administer-cluster/developing-cloud-controller-manager.md (hunk 2 above)
Comment (tengqm) [reply content/zh/docs/tasks/administer-cluster/developing-cloud-controller-manager.md:82]: 对于这类情况，第一次给出英文就好了，没有必要反复重复。

Comment (ydFu) [reply content/zh/docs/tasks/administer-cluster/developing-cloud-controller-manager.md:82]: I see. I'll keep an eye out for this next time I make a code review.

**Comment by ydFu** [comment]
/lgtm
LGTM,thank you for guiding me on the code view details for localization.

**Pull request merged**